.changeset/silver-tools-collect.md

```diff
@@ -0,0 +1,20 @@
+---
+"effect": patch
```

Comment:
```suggestion
"effect": minor
```

But too late now lol.

Reply:
Oops. Sorry. Why though? It's not breaking.

Reply:
New features are minor for semver (once API stable)

Reply:
Gotcha! If the package is versioned below 1.0.0 does patch for feature still apply (e.g. the vitest package)?